### ISSUE TITLE
Clip model size is too small

### ISSUE THREAD

**Issue by dwsmart32** [OpenGVLab/InternVideo#114]
Hello, really appreciate your great work.

I checked that you guys wrote "We also learn a CLIP-style InternVideo2 indicated by InternVideo2clip. It is post-pretrained from InternVideo2s2 by only preserving video and text encoders and contrastive loss." in your paper.

But I found out that this model [InternVideo2-CLIP-1B-224p-f8] in huggingface is too small just a few MB. And according to before issue I noticed that that pth file in huggimgface is add on parameter not a full parameter.

1. So as i understood, there might be only clip model that post trained after stage2 right? 
2. I want to know  how can I initialize that clip model and utilize. I want to get clip score from that model. It would be really grateful if you let me know exact way to do that.

Thank you in advance.

**Comment by Andy1621** [comment]
For your question, we only finetuned the `AttentionPool` in the vision encoder for `CLIP` model. And the main parameters are not updated.

Please check the zero-shot evaluation code for `CLIP` to load the model.

**Comment by dwsmart32** [comment]
Thanks for your reply. Then you mean I can use clip when at least two components get ready which are Internvideo2-s2 parameter(main parameter which has not been updated) and Internvideo2-clip(additional small parameter), right?

It would be really grateful if you let me know when are you guys going to update main parameter approximately.

I m looking forward to utilize your model to my work.

Appreciate for your great work once again. @Andy1621

**Comment by heidonomm** [comment]
Hey @Andy1621 👋 
Is there any plan to made the model easily loadable through Huggingface as well?
I'm finding it quite error-prone following the [setup_model](https://github.com/OpenGVLab/InternVideo/blob/main/InternVideo2/multi_modality/tasks_clip/shared_utils.py#L40) in the scripts folder.

Would be very interested in using the model in my work :)
Thanks.
